README.md

```diff
@@ -12,43 +10,125 @@ This package allows Python developers to write software that makes use of the We
 )
+patient = client.Patient(
+  firstName="Foo",
+  lastName="Bar",
+).create()
+```
+
+## Examples
+
+### Create a Patient
+
+```python
+patient = client.Patient(
+  firstName="Peter",
+  lastName="Parker"
+).create()
+```
 
+### Get and Update an existing Patient
 
-### Patient methods
-patient = welkin.Patient(firstName="Foo", lastName="Bar").create()  # Create
+```python
+# single patient
+patient = client.Patient(
+  id=patient_id
+).get()
 
-patient = welkin.Patient(id="6801d498-26f4-4aee-961b-5daffcf193c8").get()  # Read
-patients = welkin.Patients().get()  # Read all/list
+patient.update(firstName="Miles")
+patient.delete()
+
+# list of patients
+patients = client.Patients(filter=**filter_kwargs).get()
+```
 
-patient.update(firstName="Baz")  # Update
-patient.delete()  # Delete
+### Update the CDT fields on an existing Patient
 
-### User methods
+```python
+patient = client.Patient(id=patient_id).get()
+possible_cdts = patient.CDTs(name="Initial Checkup").get()
+found_cdt = possible_cdts[0]
+
+# associate the cdt with the patient in memory
+# and pull any additional details from welkin
+cdt = patient.CDT(**found_cdt).get()
+cdt.update(**{
+    "cdtf-weight": 190.2,
+})
+```
+
+### Create a User
+
+```python
 user = client.User(username="bar", email="bar@foo.com").create()  # Create
+```
+
+### Get or search for a user
+
+```python
 
-user = welkin.User(id="301b2895-cbf0-4cac-b4cf-1d082faee95c").get()  # Read
-users = welkin.Users().get()  # Read all/list
-uasers = welkin.Users().get(
-    search="lightmatter", region="east-coast", seat_assigned=True, user_state="ACTIVE"
-)  # Filtered read all/list
+# get a single user
+user = client.User(id=user_id).get()
+
+# get a list of users
+users = client.Users().get()  # Read all/list
+
+# filter/search for a specific subset of user(s)
+uasers = client.Users().get(
+    search="lightmatter",
+    region="east-coast",
+    seat_assigned=True,
+    user_state="ACTIVE"
+)
+```
+
+### Update or delete a user
 
-user.update(firstName="Baz")  # Update
-user.delete()  # Delete
 ```
+user.update(firstName="Spiddy")
+user.delete()
+```
+
+## Common Issues
+
+1. I keep getting errors about `self` not having an `id`, or `self._parent` being `None`. Help!
+This generally means you are trying to perform an action on an object which was pulled from
+welkin but hasn't been properly wrapped by it's parent. This is common when pulling a list of objects
+and then trying to perform operations on an object in that list.
+
+The easiest workaround is to initialize a new value in memory that is associated with the parent.
+For example, change this:
+
+```python
+cdt = patient.CDTs(name="Initial Checkup").get()[0]
+cdt.update(**fields)
+```
+
+Into this:
+
+```python
+found_cdt = patient.CDTs(name="Initial Checkup").get()[0]
+cdt = patient.CDT(**found_cdt).get()
+cdt.update(**fields)
+```
+To understand why this happens ref: [`SchemaBase`](https://github.com/Lightmatter/welkin-health/blob/main/welkin/models/base.py)
```

Comment:
This isn't something to document, but something to fix. Need a separate PR to address this behavior.